modules/desplayToPage.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/no-cycle */
+import Book from './books.js';
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] it would be better as the previous reviewers asked you should remove the disable linter lines from all the files in order to keep the code clean and pro